04-environment.ipynb

```diff
@@ -2,236 +2,204 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "id": "extra-shame",
    "metadata": {},
    "source": [
-    "# Complex computing environments\n",
-    "Now let's imagine some non-trivial tasks that require specific environments in which to compute "
+    "# A \"real life\" example"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "limiting-pension",
```

Comment:
Is this some autogenerated thing from jupyter?

Reply:
Must be, I am certainly not going to the bother of doing that!